README.md

```diff
@@ -97,12 +97,24 @@ Welcome to our source speech repository about [mobx-state-tree](https://github.c
  6. We can now use a sentence to instanciate our powerpuff, far much simpler ✨
 
 #### [#37](https://github.com/Dedetat/powerpuff-yourself/pull/37/files) - afterCreate
-> Say that we want to capitalize the name given to a Powerpuff (yes everybody can be mistaken, even better ones 😉)
+> Say we want to capitalize the name given to a Powerpuff (yes everybody can be mistaken, even better ones 😉)
  1. `afterCreate` is called after the model instanciation 💁‍♂️
  2. It should be defined as actions to access the `self` of the instance
  3. Here we simply capitalize the name and the job is done 🙌
 
 
 ### Nested models
+> With mobx-state-tree, it's possible to define models that depends on another
+
+#### [#38](https://github.com/Dedetat/powerpuff-yourself/pull/38/files) - Nest models
+> Now it's easy to create a Powerpuff, they are spreading and it's becoming a mess ! Let's create a store to bring back some order
+ 1. We create a new model of **Store**, called `Store` (yeah, still clever 🤓)
```

Comment:
...We create a new model, wich will be our centralized **Store**, called `Store`...